www/templates/series-detail.html

```diff
@@ -8,7 +8,7 @@
 <script src="/static/js/jquery.colorbox-min.js"></script>
 {% endblock %}
 
-{% block title %}{{ stripped_subject }}{% endblock %}
+{% block title %}{{ stripped_subject }}{% if has_other_revisions %} (v{{ version }}){% endif %}{% endblock %}
```

Comment:
Can we put this before the stripped_subject in brackets so it better matches the order and format in the original subject?

```
{% block title %}{% if has_other_revisions %}[v{{ version }}] {% endif %}{{ stripped_subject }}{% endblock %}
```